src/components/button/SCreateButton.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+import { IconType } from "react-icons";
+import React from "react";
+import { ButtonSize } from "../../styles/theme";
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement>{
+  label: string;
+  Icon?: IconType;
+  size?: ButtonSize;
+  borderRadius? :string;
+  fontSize: string;
+  iconSize?: number;
+}
+
+function SCreateButton({ label, size = "small", Icon, borderRadius, fontSize, iconSize = 16, ...props}: ButtonProps) {
+  return (
+    <SCreateButtonStyle size={size} borderRadius={borderRadius} fontSize={fontSize} {...props}>
+      <div className="icon">{Icon && <Icon size={iconSize} />}</div>
```

Comment:
```suggestion
{Icon && (
    <div className="icon">
      <Icon size={iconSize} />
    </div>
  )}
```
  
아이콘 없을 때 div 자체를 렌더링 안 하는게 좋을 것 같아요!